src/app/components/PortraitVideoModal/index.tsx

```diff
@@ -179,34 +180,37 @@ const PortraitVideoModal = ({
   }, []);
 
   return (
-    <div
-      role="dialog"
-      aria-modal="true"
-      aria-label={modalLabel}
-      ref={modalRef}
-      css={styles.modal}
-    >
-      <button
-        ref={closeButtonRef}
-        type="button"
-        data-testid="close-modal-button"
-        css={styles.closeButton}
-        className="focusIndicatorInvert"
-        onClick={onClose}
+    <>
+      <Global styles={{ '.body-overflow-hidden': { overflow: 'hidden' } }} />
```

Comment:
Why don't we add this to the style file like we do here: https://github.com/bbc/simorgh/blob/latest/ws-nextjs-app/pages/%5Bservice%5D/av-embeds/AvEmbedsPageLayout.tsx#L10

Reply:
Good idea. I forgot we did that there, I took this first pass from the Image component but this way means I can remove the `classList` functions as well 👍